tardis/opacities/opacity_solver.py

```diff
@@ -2,9 +2,15 @@
 from tardis.opacities.opacity_state import (
     OpacityState,
 )
+from tardis.opacities.macro_atom.macroatom_solver import (
+    MacroAtomSolver,
+)
+from tardis.opacities.macro_atom.macroatom_state import MacroAtomState
 import numpy as np
 import pandas as pd
 
+from tardis.transport.montecarlo.configuration import montecarlo_globals
+
```

Comment:
The three new imports is not used in this file, is this something preparing for the future refactor?